foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/databaseaccess/InOutputParameterForCallableStatement.java

```diff
@@ -51,11 +52,18 @@ public InOutputParameterForCallableStatement(Object inParameter, DatabaseField o
         }
     }
 
+    @Override
     public void set(DatabasePlatform platform, PreparedStatement statement, int index, AbstractSession session) throws SQLException {
         platform.setParameterValueInDatabaseCall(inParameter, statement, index, session);
         super.set(platform, statement, index, session);
     }
 
+    @Override
+    public void set(DatabasePlatform platform, CallableStatement statement, String index, AbstractSession session) throws SQLException {
```

Comment:
calling String arg here `index` looks confusing to me

Reply:
I agree, ty for noticing this. Ill go back through and change the var names to be "int parameterIndex" and "String parameterName". Does that sound good?